go.mod

```diff
@@ -17,39 +17,53 @@ require (
 // we need to use the same version of the dependencies as k6.
 replace github.com/spf13/afero => github.com/spf13/afero v1.1.2
```

Comment:
TBH, I believe we don't need this. Can we maybe clean this as part of the Sobek?